manifests/nodejs.yml

```diff
@@ -1,3 +1,34 @@
+refs:
+  - &ref_3.0.0 '>=3.0.0 || ^2.13.0'
```

Comment:
@cbeauchesne are you kidding me the YAML library doesn't support non alphanumeric chars in anchors ? xD
```
E   yaml.scanner.ScannerError: while scanning an anchor
E     in "manifests/nodejs.yml", line 2, column 5
E   expected alphabetic or numeric character, but found '.'
E     in "manifests/nodejs.yml", line 2, column 11
```

Reply:
>YAML anchors and aliases cannot contain the ' [ ', ' ] ', ' { ', ' } ', and ' , ' characters.

So is the yaml parser used currently not following the spec ?

Reply:
I'm using [the most used yaml parser](https://pyyaml.org/)

Is it something blocking ?

Reply:
I checked the source code and the only characters allowed are `a-zA-Z0-9-_` which is annoyingly and non-spec-following restrictive. So what would be the alternative ? `&ref_5_6_0`, `&ref_5-6-0` ?